### PR TITLE
[No QA] Convert doc to .md for testing MacOS and iOS apps on windows/linux

### PR DESCRIPTION
**Pull request by rushatgabhane**
cc: @mallenexpensify 

### Details

This is the final step of https://github.com/Expensify/App/issues/7485#issuecomment-1029308825. This PR converts this [doc](https://docs.google.com/document/d/1TD8A-SQbBN2EsnF98cEib2wiERK68fux2P0LGSkobrg/edit#) to a `.md` file.

You can checkout the end result over here -https://github.com/rushatgabhane/Expensify-App/blob/fix-7485/TESTING_MACOS_AND_IOS.md

### Fixed Issues

$ https://github.com/Expensify/App/issues/7485

### CODE REVIEW

**Comment by rushatgabhane** [comment]
I'm not a fan of the file name `TESTING_MACOS_AND_IOS.md`, but can't really think of something elegant.

Also, should I keep the alternative solution section at the end?

**Comment by mallenexpensify** [comment]
@rushatgabhane , this looks great! Nice, clean, not too wordy.  Can you add one thing? We want to direct contributors to #expensify-open-source if they run into issues, have questions or propose updates/improvements.  Other than than 👍

**Comment by rushatgabhane** [comment]
Great call, added this on top of the file -

If you need help, have any questions or suggestions, please ask in the [`#expensify-open-source` Slack channel](https://github.com/Expensify/App/blob/main/CONTRIBUTING.md#asking-questions)

File: TESTING_MACOS_AND_IOS.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+Majority of the contributors need MacOS for short bursts of time for the purpose of testing. Cloud services that offer MacOS instances like AWS, MacInCloud, MacStadium etc have a very high minimum cost. [Scaleway’s M1 Mac minis](https://www.scaleway.com/en/hello-m1/) provide dedicated instances at just $2.7 per day.
+
+
+| **Cloud provider**|**Minimum cost** |
+| ----------- | ----------- |
+| **Scaleway** | **$2.7 per day 🥇** |
+| MacInCloud |$19 for 5 days, or **$35 per month** |
+| MacStadium | $132 per month |
+| AWS | $26 per day |
+
```
Comment (neil-marcellini) [inline TESTING_MACOS_AND_IOS.md:18]: I think we should remove these lines because this document is now intended to serve as a quick set up guide. The reader already knows that they need to test on iOS and MacOS. Perhaps we could leave the solution portion for more context about why we chose Scaleway, but I think that is probably unnecessary.

File: TESTING_MACOS_AND_IOS.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+source  ~/.zshrc
+
+# Install cocoapods
+sudo gem install ffi -- --enable-system-libffi
+brew install cocoapods
+```
+
+You now have a dedicated Mac machine running 🎉
+
+Clipboard is shared with your local machine, and it’s crazy fast and responsive.
```
Comment (neil-marcellini) [inline TESTING_MACOS_AND_IOS.md:72]:
```suggestion
The clipboard is shared with your local machine, and it’s crazy fast and responsive.
```

File: TESTING_MACOS_AND_IOS.md [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+
+## Alternative solution
+Virtual machines for MacOS like [macOS-Simple-KVM](https://github.com/foxlet/macOS-Simple-KVM) are the easiest and most accessible way for a contributor to test the app on these platforms. But it violates Apple’s EULA, and isn’t legal.
```
Comment (neil-marcellini) [inline TESTING_MACOS_AND_IOS.md:75]: I also think that this section is not relevant in a set up guide, especially because this option isn't legal.

**Review by neil-marcellini — CHANGES_REQUESTED** [review]

File: TESTING_MACOS_AND_IOS.md (hunk 3 above)
Comment (rushatgabhane) [reply TESTING_MACOS_AND_IOS.md:75]: 👍 removed

File: TESTING_MACOS_AND_IOS.md (hunk 1 above)
Comment (rushatgabhane) [reply TESTING_MACOS_AND_IOS.md:18]: Yeah, makes sense. I think we should keep the solution part tho.
Because I'm sure contributors will explore other cloud providers and this will just save them some time.

**Review by neil-marcellini — CHANGES_REQUESTED** [review]
This is looking great. Here are two small tweaks, and then it will be ready to go.

File: TESTING_MACOS_AND_IOS.md [hunk 4]
```diff
@@ -0,0 +1,67 @@
+# Test MacOS and iOS app on windows/linux
+
+If you need help, have any questions or suggestions, please ask in the [`#expensify-open-source` Slack channel](https://github.com/Expensify/App/blob/main/CONTRIBUTING.md#asking-questions).
+
+Majority of the contributors need MacOS for short bursts of time for the purpose of testing. Cloud services that offer MacOS instances like AWS, MacInCloud, MacStadium etc have a very high minimum cost. [Scaleway’s M1 Mac minis](https://www.scaleway.com/en/hello-m1/) provide dedicated instances at just $2.7 per day.
```
[the remark this answers is not in the document]
Comment (rushatgabhane) [reply TESTING_MACOS_AND_IOS.md]: Thanks!

**Review by neil-marcellini — APPROVED** [review]
Looks good!
